cmd/limactl/start.go

```diff
@@ -92,9 +94,16 @@ func loadOrCreateInstance(cmd *cobra.Command, args []string) (*store.Instance, e
 
 	if ok, u := argSeemsTemplateURL(arg); ok {
 		templateName := u.Host
+		if u.Path != "" {
+			templateName, err = securejoin.SecureJoin(templateName, u.Path)
+			if err != nil {
+				return nil, err
+			}
+		}
```

Comment:
It doesn't make sense to use `SecureJoin` here because it is still a relative path. And `filepath.Join` will just ignore empty elements, so no need for the conditional either:

```suggestion
		templateName = filepath.Join(u.Host, u.Path)
```

The real `SecureJoin` check happens later in `readTemplate` when this relative name is joined with the full path to the examples directory.